website/project/metadata/schemas.py

```diff
@@ -24,5 +18,8 @@ def from_json(fname):
     ensure_schema_structure(from_json('osf-standard-1.json')),
```

Comment:
I agree - done in 8a82320

Note that you can return inside the context manager, eliminating a variable assignment:

```python
with open(os.path.join(here, fname)) as f:
    return json.load(f)
```